molecule/enable_libpodstats/verify.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- import_playbook: ../common/verify.yml
+
+- name: "Verify collectd exec plugin"
```

Comment:
collectd libpodstats is not run from the collectd exec plugin but runs as its own native plugin

Reply:
> I noticed that this role puts the following in the configuration for the additional types.db:
> 
> ```
> TypesDB "/usr/share/collectd/types.db"
> TypesDB "/etc/collectd.d/types.db.libpodstats"
> ```
> 
> It has been a while, but I am fairly certain collectd only accepts the second one which would break the functionality of all the other plugins. I think I tried the same thing when I was making the puppet changes. It should look like:
> 
> ```
> TypesDB "/usr/share/collectd/types.db" "/usr/share/collectd/types.db.libpodstats"
> ```

Docs indicate that multiple TypesDB can be specified over multiple lines (or in a single line)[1], and I took that option since it was easier to format, and easier to grok/debug. I tested it on my side by splitting some definitions out into separate files and verified that multiple files work.

[1] https://collectd.org/documentation/manpages/types.db.5.shtml

Reply:
Interesting, I don't know why I did it that way then